### PR TITLE
Feat: add nvm solution in node vignette

### LINKED ISSUE #345: RStudio ignores Node.js version set by `nvm`
In RStudio, Node commands are not using Node version set by `nvm`. This works fine when the R session is started in the terminal (both `R` and `radian`):
```
r$> rhino::diagnostics()
Linux 5.15.0-41-generic #44-Ubuntu SMP Wed Jun 22 14:20:53 UTC 2022
R version 4.2.1 (2022-06-23)
rhino: 1.1.0
node: v16.16.0
yarn: 1.22.19
```
 and fails in RStudio R console:
```
> rhino::diagnostics()
Linux 5.15.0-41-generic #44-Ubuntu SMP Wed Jun 22 14:20:53 UTC 2022
R version 4.2.1 (2022-06-23)
rhino: 1.1.0
node: v14.19.3
yarn: 1.22.19
```

### PR DESCRIPTION
**Pull request by Leszek-Sieminski**
### Changes

- [x] Added a section about `nvm` in the Node vignette
- [x] Added the solutions for Windows, Mac and Linux

Closes #345

### How to test
Read the vignette

### CODE REVIEW

File: vignettes/explanation/node-js-javascript-and-sass-tools.Rmd [hunk 1]
```diff
@@ -2,35 +2,78 @@
 title: "Explanation: Node.js - JavaScript and Sass tools"
 output: rmarkdown::html_vignette
 vignette: >
+  %\VignetteEncoding{UTF-8}
   %\VignetteIndexEntry{Explanation: Node.js - JavaScript and Sass tools}
   %\VignetteEngine{knitr::rmarkdown}
-  %\VignetteEncoding{UTF-8}
+editor_options: 
+  markdown: 
+    wrap: 72
```
Comment (kamilzyla) [inline vignettes/explanation/node-js-javascript-and-sass-tools.Rmd:10]: Do we need this? Also, we wrap around column 100 in general.

File: inst/WORDLIST [hunk 2]
```diff
@@ -42,3 +42,5 @@ usethis
 webpack
 yml
 Żyła
+nvm
+PowerShell
```
Comment (kamilzyla) [inline inst/WORDLIST:46]: Did you use `spelling::update_wordlist()` to update it? I'm surprised that these landed out of alphabetical order.

File: vignettes/explanation/node-js-javascript-and-sass-tools.Rmd [hunk 3]
```diff
@@ -2,35 +2,78 @@
...
+Open your terminal of choice (i.e. Bash) and run
+```
+rstudio
+```
+
+#### Windows
+Open your Windows terminal of choice (i.e. Terminal, PowerShell, Git Bash) and
+run:
+```
+
```
Comment (kamilzyla) [inline vignettes/explanation/node-js-javascript-and-sass-tools.Rmd:61]: Something is missing here :wink:

File: vignettes/explanation/node-js-javascript-and-sass-tools.Rmd [hunk 4]
```diff
@@ -2,35 +2,78 @@
...
+4.  `lint_sass()`
+5.  `test_e2e()`
+
+## Node directory
+
+Under the hood Rhino will create a `.rhino/node` directory in your
+project to store the specific libraries needed by these tools. This
+directory is git-ignored by default and safe to remove.
+
+## Details
```
Comment (kamilzyla) [inline vignettes/explanation/node-js-javascript-and-sass-tools.Rmd:38]: I'd suggest just three subsections (header level `###`) here:
1. Node directory (move from above).
2. Node installation via nvm.
3. Building Sass.

File: vignettes/explanation/node-js-javascript-and-sass-tools.Rmd [hunk 5]
```diff
@@ -2,35 +2,78 @@
...
-This directory is git-ignored by default and safe to remove.
-
-The `build_sass()` function is worth an additional comment.
-Depending on the configuration in `rhino.yml`
-it can use either the [sass](https://www.npmjs.com/package/sass) Node.js package
-or the [sass](https://rstudio.github.io/sass/) R package.
-We recommend the Node.js version, as it is the primary, actively developed implementation of Sass.
-In contrast, the R package uses the deprecated
-[LibSass](https://sass-lang.com/blog/libsass-is-deprecated) implementation.
+## About
```
Comment (kamilzyla) [inline vignettes/explanation/node-js-javascript-and-sass-tools.Rmd:13]: The "About" and "Usage" are very short; I'd merge them into one section.

**Review by kamilzyla — APPROVED** [review]
LGTM

File: inst/WORDLIST (hunk 2 above)
Comment (Leszek-Sieminski) [reply inst/WORDLIST:46]: Ah, no I didn't. Honestly assumed that it was meant to be edited by hand, I will fix this

File: vignettes/explanation/node-js-javascript-and-sass-tools.Rmd (hunk 1 above)
Comment (Leszek-Sieminski) [reply vignettes/explanation/node-js-javascript-and-sass-tools.Rmd:10]: This was generated automatically by my RStudio options, I usually use wrapping around 80. I will remove those lines

File: vignettes/explanation/node-js-javascript-and-sass-tools.Rmd (hunk 5 above)
Comment (Leszek-Sieminski) [reply vignettes/explanation/node-js-javascript-and-sass-tools.Rmd:13]: fixed

File: vignettes/explanation/node-js-javascript-and-sass-tools.Rmd (hunk 3 above)
Comment (Leszek-Sieminski) [reply vignettes/explanation/node-js-javascript-and-sass-tools.Rmd:61]: Didn't push the change, will add this

File: vignettes/explanation/node-js-javascript-and-sass-tools.Rmd (hunk 1 above)
Comment (Leszek-Sieminski) [reply vignettes/explanation/node-js-javascript-and-sass-tools.Rmd]: Fixed

File: inst/WORDLIST (hunk 2 above)
Comment (Leszek-Sieminski) [reply inst/WORDLIST]: Fixed

File: vignettes/explanation/node-js-javascript-and-sass-tools.Rmd (hunk 3 above)
Comment (Leszek-Sieminski) [reply vignettes/explanation/node-js-javascript-and-sass-tools.Rmd]: Fixed

File: vignettes/explanation/node-js-javascript-and-sass-tools.Rmd (hunk 4 above)
Comment (Leszek-Sieminski) [reply vignettes/explanation/node-js-javascript-and-sass-tools.Rmd]: Done

**Pull request merged**